code/game/objects/items/weapons/clown_items.dm

```diff
@@ -36,9 +36,10 @@
 	cleanspeed = 10 //much faster than mop so it is useful for traitors who want to clean crime scenes
 
 /obj/item/weapon/soap/suicide_act(mob/user)
-	user.visible_message("<span class='suicide'>[user] crams the bar of soap down \his throat! It looks like \he's trying to commit suicide..</span>")
-	. = TOXLOSS
-	src.loc = user
+	user.say(";FFFFFFFFFFFFFFFFUUUUUUUDGE!!")
+	user.visible_message("<span class='suicide'>[user] lifts the [src.name] to their mouth and gnaws on it furiously, producing a thick froth! They'll never get that bb gun now!")
+	PoolOrNew(/obj/effect/effect/foam, loc)

```

Comment:
I thought about doing that; looked around and couldn't actually find the BB gun anywhere, at least not that time that I looked; can you point me towards it?